src/java.desktop/macosx/classes/sun/font/CFontManager.java

```diff
@@ -225,7 +225,7 @@ public Object run() {
             String defaultFallback = "Lucida Grande";
 
             setupLogicalFonts("Dialog", defaultFont, defaultFallback);
-            setupLogicalFonts("Serif", "Times", "Times");
+            setupLogicalFonts("Serif", "Times", "Times New Roman");
```

Comment:
Can we do it under macos >= Monterey check so that we are sure it's not affecting older releases just as do for JTabbedPane legibility issue using JRSUIUtils.isMacOSBigSurOrAbove() check?
I see we have both Times and Times New Roman in "Font Book"...BTW, did you try "Restore Standard fonts" option in Font Book to see if "Times" font somehow comes back?

Reply:
Just to clarify, I see those 2 fonts in BigSur and Catalina..